src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -165,9 +165,7 @@ def from_request(
         Returns:
             The created `PipelineRunSchema`.
         """
-        configuration = json.dumps(
-            request.pipeline_configuration, default=pydantic_encoder
-        )
+        config = request.pipeline_configuration
```

Comment:
Love how this change makes the code cleaner and easier to reason with. Will definitely use it.